Project.toml

```
@@ -26,14 +26,14 @@ TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
 TrixiBase = "9a0f1c46-06d5-4909-a5a3-ce25d3fa3284"
 
 [compat]
-BandedMatrices = "1"
+BandedMatrices = "1.5"
 DiffEqBase = "6.157"
 FastBroadcast = "0.3.5"
 ForwardDiff = "0.10.36"
 Interpolations = "0.15.1"
 LinearAlgebra = "1"
 PolynomialBases = "0.4.15"
-PreallocationTools = "0.4.14"
+PreallocationTools = "0.4.27"
```

```suggestion
PreallocationTools = "0.4.23"
```

This is what https://github.com/SciML/OrdinaryDiffEq.jl/blame/master/lib/OrdinaryDiffEqNonlinearSolve/Project.toml requires